parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ColumnChunkPageWriteStore.java

```diff
@@ -262,4 +262,9 @@ public void flushToFileWriter(ParquetFileWriter writer) throws IOException {
     }
   }
 
+  public void flushToFileWriter(ColumnDescriptor path, ParquetFileWriter writer) throws IOException {
```

Comment:
If I'm not mistaken, the visibility of this method can be restricted to package private